benchmarks/test-file-overhead/README.md

```diff
@@ -0,0 +1,6 @@
+First, run `./prepare.sh` to generate the benchmark files. On Windows, use a Bash (WSL, Git, Cygwinm ...) to do this, but you can use CMD for the actual benchmark run if the CMD environment is what you want to benchmark for.
+
+To run the benchmark, use a benchmarking tool such as [hyperfine](https://github.com/sharkdp/hyperfine):
+```sh
+hyperfine -w 3 -m 10 ../../jest /tmp/other-jest-clone-to-compare-against/jest
```

Comment:
what numbers are you seeing with this PR, btw?

Reply:
https://github.com/facebook/jest/issues/7963#issuecomment-732143757 this sums it up, on Windows `master` is roughly 50% faster on many cheap test files than 26, with this it's roughly 70% faster on many cheap test files than 26.